doc/guide/install.rst

```diff
@@ -224,11 +238,11 @@ The CLIMADA Python package defines the following `extras <https://peps.python.or
      - combination of ``doc`` and ``test``
 
 For executing the pre-defined test scripts in exactly the same way as they are executed by the automated CI pipeline, you will need ``make`` to be installed.
-On macOS and on Linux it is pre-installed. On Windows, it can easily be installed with conda:
+On macOS and on Linux it is pre-installed. On Windows, it can easily be installed with Conda:
```

Comment:
Just to be sure: do we still need the `make` instructions?

Reply:
@chahank Can we resolve this?